### DISCUSSION TITLE
void** doesn't accept Any**

### DISCUSSION THREAD

**Discussion by SupinePandora43** [dotnet/csharplang#6150] [General]
```cs
byte b = 10;
byte* bPtr = &b;
byte** bPtrPtr = &bPtr;

void* voidPtr = bPtr;
void** voidPtrPtr = bPtrPtr; // error: CS0266
```

**Comment by jnm2** [comment]
You need `(void**)bPtrPtr` because it's not an upcast. (It's a downcast from the intermediate `void*`.) You don't need an explicit `(void*)` because that's an upcast.
